src/navigation/Nav.jsx

```diff
@@ -304,19 +305,21 @@ export class Nav extends React.Component {
 				}),
 				icon: (
 					<Flex noGutters align="center">
-						<FlexItem>
-							<AvatarMember small member={self} />
-						</FlexItem>
-						<FlexItem
-							shrink
-							className="display--none atMedium_display--block"
-						>
-							<Icon
-								shape="chevron-down"
-								size="xxs"
-								className="padding--left-half"
-							/>
-						</FlexItem>
+						<Button aria-haspopup reset aria-label={profile.label}>
```

Comment:
This `button` element is actually nested in a `button` created by `NavItem.jsx`, which results in an invalid DOM that prints a warning in dev and causes a pretty major UI layout issue on slow connections (https://meetup.atlassian.net/browse/MG-3133). The `button` created by `NavItem` already has `aria-haspopup`, so I've pushed an update that removes this `<Button />` wrapper and passes `aria-label` directly to the `Flex` wrapper, which should address the voiceover a11y concern